dataspace-connector/full/ui/ui.env

```diff
@@ -1,3 +1,3 @@
-CONFIGMANAGER_URL=http://configmanager:8081
+CONNECTOR_URL=https://dataspaceconnector:8080
 UI_BACKEND_URL=http://localhost:8083
-UI_TITLE=IDS Configuration Manager
\ No newline at end of file
+UI_TITLE=Dataspace Connector UI
```

Comment:
Missing empty line at the end of the file. Maybe adding a editorconfig would help?